core/tests/unit/ldml/kmx_plus.tests.cpp

```diff
@@ -9,12 +10,13 @@
 // needed for streaming operators
 #include "utfcodec.hpp"
 
+#ifdef _MSC_VER
+#pragma execution_character_set("utf-8")
+#endif
```

Comment:
Is it possible for this to live in meson.build flags?

Reply:
I can look into it, I wanted to first see if this can be fixed at a localized level.

Note that on the Mac, invalid non-characters produced a similar error, which is why I am using hex arrays for those. 

Also, this is only in test code, if we do set it as a flag, I might localize it to just this project. In case there are any ripple effects…

Reply:
sadface:

```
../../../tests/unit/ldml/kmx_plus.tests.cpp(14): error C3437: #pragma execution_character_set is not supported when /source-charset, /execution-charset, or /utf-8 has been specified
```

Reply:
```
 "/source-charset:utf-8"
```

is already being passed